### ISSUE TITLE
Naming collision with `nexus` on global interfaces on `inputTypes`

### ISSUE THREAD

**Issue by nikolasburk** [graphql-nexus/nexus-plugin-prisma#82]
This can be reprocuded using the `graphql` example from the `prisma-examples` by removing the `skipLibCheck` option from tsconfig.json.

```
$ yarn tsc
yarn run v1.13.0
warning package.json: No license field
$ /Users/nikolasburk/Desktop/prisma-examples/typescript/graphql/node_modules/.bin/tsc
node_modules/nexus-prisma/dist/definitions/inputObjectType.d.ts:13:93 - error TS2344: Type 'string' does not satisfy the constraint '"UserWhereUniqueInput" | "PostWhereInput" | "UserWhereInput" | "PostWhereUniqueInput" | "UserCreateInput" | "PostCreateManyWithoutAuthorInput" | "PostCreateWithoutAuthorInput" | ... 18 more ... | "PostSubscriptionWhereInput"'.

13 export declare function prismaInputObjectType<TypeName extends PrismaInputObjectTypeNames = string>(typeConfig: PrismaInputObjectTypeConfig<TypeName>): core.NexusWrappedType<core.NexusInputObjectTypeDef<TypeName>>;
                                                                                               ~~~~~~

node_modules/nexus-prisma/dist/definitions/objectType.d.ts:15:83 - error TS2344: Type 'string' does not satisfy the constraint '"Query" | "Mutation" | "Subscription" | "Post" | "User" | "UserConnection" | "PageInfo" | "UserEdge" | "AggregateUser" | "PostConnection" | "PostEdge" | "AggregatePost" | "BatchPayload" | "UserSubscriptionPayload" | "UserPreviousValues" | "PostSubscriptionPayload" | "PostPreviousValues"'.

15 export declare function prismaObjectType<TypeName extends PrismaObjectTypeNames = string>(typeConfig: PrismaObjectTypeConfig<TypeName>): core.NexusWrappedType<core.NexusObjectTypeDef<TypeName>>;
                                                                                     ~~~~~~

node_modules/nexus/dist/typegenTypeHelpers.d.ts:3:13 - error TS2320: Interface 'NexusGen' cannot simultaneously extend types 'NexusPrismaTypes' and 'NexusGenTypes'.
  Named property 'inputTypes' of types 'NexusPrismaTypes' and 'NexusGenTypes' are not identical.

3   interface NexusGen {}
              ~~~~~~~~


Found 3 errors.

error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**Comment by Weakky** [comment]
This is fixed under `nexus-prisma@0.1.5` and `nexus-prisma-generate@0.1.4`

**Note**: This is a breaking change as `nexus-prisma` now uses its own global interface called `NexusPrismaGen`. **You therefore need to re-run `nexus-prisma-generate`**

**Comment by 5achinJani** [comment]
I'm getting this error outta nowhere:
![image](https://user-images.githubusercontent.com/23077544/98957743-89c0d380-2527-11eb-9dda-e11a872ae5a8.png)

It's is weird. The root cause is by importing this NexusGenInputs . If I disable that line then it works.

![image](https://user-images.githubusercontent.com/23077544/98958042-e0c6a880-2527-11eb-8668-f89791e01b07.png)

I don't get this error when running through `ts-node-dev` it throws error only when I run `tsc`. 

```json
    "nexus": "0.11.7",
    "nexus-prisma": "0.3.8",
    "prisma-client-lib": "1.34.10"
```

I tried running the `npx nexus-prisma-generate` but still I get this error. 
I've also made sure to disable the type check for src/generated and "skipLibCheck": true,
tsconfig
```json
{
  "compilerOptions": {
    "sourceMap": true,
    "outDir": "dist",
    "lib": ["esnext", "dom"],
    // "strict": true, // `strict` is commented because of this issue: https://github.com/prisma/prisma/issues/3774; all its options except `strictNullChecks` & `strictPropertyInitialization` are explicitly set below.
    "noImplicitAny": false,
    "noImplicitThis": true,
    "alwaysStrict": true,
    "strictBindCallApply": true,
    "strictFunctionTypes": true,
    "skipLibCheck": true, // `skipLibCheck` is enabled until this is issue is fixed: https://github.com/prisma/nexus-prisma/issues/82
    "resolveJsonModule": true
  },
  "exclude": ["src/generated"]
}

```
